src/components/addBlockModal/useAddBlock.ts

```
@@ -100,7 +100,7 @@ function useAddBlock(): AddBlock {
       // Add the block to a copy of the extension
       const newBlock = await makeNewBlock(block);
       const newExtension = produce(activeExtension, (draft) => {
-        const pipeline = get(draft, addBlockLocation.path);
```

These `get` and `getIn` calls technically aren't safe from a null checks standpoint, unless `path` is guaranteed to exist.

Let me know if they should all be followed by  `assertNonNullish`

---

@BLoe you probably have the most context here. What are your thoughts?

---

I would think we would _want_ an error here, so I would lean towards `assertNonNullish`. 

Alternatively we could use the third arg for `get` to assign a default value, but that doesn't seem like the right decision here.

---

Yeah, I think we would want to throw an error if this doesn't return a valid pipeline, but I doubt that's technically possible right now.

---

Actually all of these already do throw an NPE because they're all followed by `pipeline.splice`. 

However in practice I think the "path" is guaranteed to exist and can't lead to `null` (unless the path generation itself is incorrect)